### ISSUE TITLE
Doesn't interact with Helicopters

### ISSUE THREAD

**Issue by jarquafelmu** [folknor/factorio-fill#1]
[Helicopters#10](https://github.com/kumpuu/factorio-helicopters/issues/10)

**Comment by folknor** [comment]
Yeah, this doesn't make any sense. I specifically wrote factorio-fill to "just work". Just from doing a quick skim of the code, I can't see the problem.

Which means I'll have to load the addons up ingame and debug.

**Comment by folknor** [comment]
Closing as stale - this has been open since 2018 and the Helicopters mod is likely no longer relevant.
